Doc/library/concurrent.futures.rst

```diff
@@ -227,6 +229,54 @@ ThreadPoolExecutor Example
                print('%r page is %d bytes' % (url, len(data)))
 
 
+InterpreterPoolExecutor
+-----------------------
+
+The :class:`InterpreterPoolExecutor` class is a :class:`ThreadPoolExecutor`
+subclass that uses a pool of isolated interpreters to execute calls
+asynchronously.  Each interpreter has its own GIL, which allows the
+executor to side-step the :term:`Global Interpreter Lock
+<global interpreter lock>`.  Interpreters mostly can't share objects
```

Comment:
I think it's worth noting here *why* getting around the GIL is a good thing. In my experience, many Python devs don't fully understand what the GIL does to their threads.
```suggestion
<global interpreter lock>`, allowing the use of multiple cores.  Interpreters mostly can't share objects
```